responder/statics.py

```diff
@@ -1,5 +1,6 @@
+API_THEMES = ["elements", "rapidoc", "redoc", "swagger_ui"]
 DEFAULT_ENCODING = "utf-8"
-DEFAULT_API_THEME = "swaggerui"
+DEFAULT_API_THEME = "swagger_ui"
```

Comment:
and this variable `DEFAULT_OPENAPI_THEME`

Reply:
1107f5d039efdd2f65 also renames to `DEFAULT_OPENAPI_THEME`, see comment above.